src/cvex/linuxvm.py

```diff
@@ -56,7 +56,7 @@ def get_ansible_inventory(self) -> Path:
                     f"ansible_port={self.vag.port()} "
                     f"ansible_user={self.vag.user()} "
                     f"ansible_ssh_private_key_file={self.vag.keyfile()} "
-                    f"ansible_ssh_common_args='-o StrictHostKeyChecking=no'")
+                    f"ansible_ssh_common_args='-o StrictHostKeyChecking=no -oHostKeyAlgorithms=+ssh-rsa'")
```

Comment:
It doesn't seem to be enough.  I had to add also these parameters to overcome SSH connection issues:
`-o PubkeyAcceptedKeyTypes=+ssh-rsa -o UserKnownHostsFile=/dev/null`

Reply:
> $ ansible --version
> ansible [core 2.17.1]


> $ apt list | grep ansible
> 
> WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
> 
> ansible-core/jammy-updates,jammy-updates 2.12.0-1ubuntu0.1 all
> ansible-lint/jammy,jammy 5.4.0-2 all
> ansible-mitogen/jammy,jammy 0.3.1-3 all
> ansible/jammy,jammy 2.10.7+merged+base+2.10.8+dfsg-1 all
> python-ansible-runner-doc/jammy,jammy 2.1.1-1 all
> python-networking-ansible-doc/jammy,jammy 17.0.0-2 all
> python3-ansible-runner/jammy,jammy 2.1.1-1 all
> python3-networking-ansible/jammy,jammy 17.0.0-2 all

Reply:
Ok, it appears Ansible 2.17 dropped Python 2.7. So we would need some other process to bootstrap this system. This actually seems like a rather significant limitation for reproducibility - for instance as Ubuntu 22 becomes out of date.

Reply:
I see. We may need our own custom version of Debian 7 box with Python3 pre-installed in it. I'll take care of that.